models/user.go

```diff
@@ -28,6 +28,11 @@ type User struct {
 	Free                   bool    `gorm:"type:boolean"`
 	Credits                float64 `gorm:"type:float;default:0"`
 	CustomerObjectHash     string  `gorm:"type:varchar(255)"`
+	// the organization if any this user belongs to
```

Comment:
The organization this user belongs to. Otherwise, empty string.

Reply:
good idea